### DISCUSSION TITLE
Is there a JAX package for dealing with manifold(hyperbolic)-related models?

### DISCUSSION THREAD

**Discussion by nasyxx** [jax-ml/jax#14763] [Q&A]
Such as the Hyperbolic neural network or Riemannian adam optimizer.

**Comment by adam-hartshorne** [comment]
Is this the sort of thing you are looking for?

https://github.com/SaitejaUtpala/rieoptax

**Comment by jakubMitura14** [comment]
Geomstats also have jax backend , if you are into riemanian statistics worth exploring

**Comment by nasyxx** [comment]
> Is this the sort of thing you are looking for?
> 
> 
> 
> https://github.com/SaitejaUtpala/rieoptax

Thanks!  Almost yes.  It's worthy of reference.

**Comment by nasyxx** [comment]
> Geomstats also have jax backend , if you are into riemanian statistics worth exploring

Thanks, but it seems the JAX backend for geomstata is still work in progress.
